docs/developer/protocol/commander_core.md

```diff
@@ -0,0 +1,206 @@
+# Corsair Commander Core Protocol
+
+### Compatible devices
+
+| Device Name | USB ID | LED channels | Fan channels | Temperature channels | 
+|:-----------:|:------:|:------------:|:------------:|:--------------------:|
+| Commander Pro | `1B1C:0C1C` | 7 | 6 | 2 |
+**NOTE: The first two LED and temperature channels go to the EXT port and potentially the AIO**
+
+### Command formats
+
+**NOTES:**
+ - The commander core works in different modes so ensure the proper mode has been sent for each command
```

Comment:
is commander core supposed to be all lowercase?

Reply:
I missed this; now fixed in 5534d7eda911.